kolibri/core/analytics/pskolibri/_psposix.py

```diff
@@ -0,0 +1,34 @@
+"""Routines common to all posix systems."""
+from __future__ import absolute_import
+from __future__ import division
+
+import errno
+import os
+
+
+def pid_exists(pid):
```

Comment:
This implementation already exists in `kolibri.utils.system`

Reply:
This is a general function, not for Kolibri, but for any process. It's needed when Django commands are executed out of the Kolibri server.